.github/workflows/indigo-ci.yaml

```diff
@@ -705,18 +705,39 @@ jobs:
       - name: Git fetch tags
         run: git fetch --tags -f
       - name: Create folder for native libs
-        run: mkdir dist
+        run: mkdir -p dist/lib
       - name: Download native shared libraries
         uses: actions/download-artifact@v2
         with:
           name: indigo-native-shared-libs
+          path: dist/lib
+      - name: Download Python wheels
+        uses: actions/download-artifact@v2
+        with:
+          name: indigo-python
+          path: dist/
+      - name: Download Java jars
+        uses: actions/download-artifact@v2
+        with:
+          name: indigo-java
+          path: dist/

```

Comment:
We don't need to download jars, only native libraries

Reply:
fixed